test/e2e/service.go

```diff
@@ -1731,7 +1740,8 @@ metadata:
 		ginkgo.By("Test sevice connectivity after changing MTU on the intermediate node")
 		gomega.Eventually(func() error {
 			return buildAndRunCommand(fmt.Sprintf("sudo ip netns exec client curl %s:%d/big.iso -o big.iso", svcIPforCurl, endpointHTTPPort))
-		}, 5*time.Second).Should(gomega.BeNil(), "failed to connect with external load balancer service after changing mtu size")
+		}, 10*time.Second).Should(gomega.BeNil(), "failed to connect with external load balancer service after changing mtu size")
```

Comment:
10s timeout is not required here as that was only needed previously for network config (ns and ip configs) to complete.